### PR TITLE
readme: add evaluate snippet

### PR DESCRIPTION
**Pull request by ebidel**
pretty important method that we don't have a prominent example for.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,32 @@ const puppeteer = require('puppeteer');
...
+```js
+const puppeteer = require('puppeteer');
+
+(async () => {
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+  await page.goto('https://example.com');
+  
+  // Get the viewport of the page.
+  const viewport = await page.evaluate(() => {
```
Comment (aslushnikov) [inline README.md]: I'd rename s/viewport/dimensions to avoid confusion with `page.viewport()` method

**Review by aslushnikov — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ebidel) [reply README.md]: donedone

File: README.md [hunk 2]
```diff
@@ -70,6 +70,32 @@ const puppeteer = require('puppeteer');
...
+      height: document.documentElement.clientHeight,
+      deviceScaleFactor: window.devicePixelRatio
+    };
+  });
+  
+  console.log('Dimensions:', dimensions);
+
+  browser.close();
+})();
+
```
Comment (vsemozhetbyt) [inline README.md:96]: Missing backticks.

Comment (vsemozhetbyt) [reply README.md:96]: PR: https://github.com/GoogleChrome/puppeteer/pull/525

**Pull request merged**